Guide/src/dev_guide/getting_started/linux.md

```diff
@@ -43,6 +43,23 @@ over to it.
 
 Please follow the [official instructions](https://www.rust-lang.org/tools/install) to do so.
 
+## \[Linux] Additional Dependencies
```

Comment:
there was intentionality behind having this section live in `build_openvmm` rather than in `getting_started/linux`. 

Namely: these deps are only needed when building openvmm, but _not_ openhcl. build-igvm will make sure all necessary apt commands are invoked as part of the build... but since there is no equivalent tool for openvmm, this manual invocation is required. (tihs is what the restore-packages comment alluded to).

I might look into reverting this move in a future PR, and rewriting some of the comments to better communicate intent behind this organization